js-api-doc/importer.d.ts

```diff
@@ -375,13 +376,14 @@ export interface Importer<sync extends 'sync' | 'async' = 'sync' | 'async'> {
  * Note that while library users can rely on the importer to resolve
  * [partials](https://sass-lang.com/documentation/at-rules/use#partials), [index
  * files](https://sass-lang.com/documentation/at-rules/use#index-files), and
- * extensions, library authors must specify the entire file path.
+ * extensions, library authors must specify the entire file path in exports.
```

Comment:
Nit: `exports` should be code font, since it refers to a field in the package.json.

Reply:
Updated in 41fc3132671d87f75175f02076e76299b318a7b7